src/main/plugin/iso19115-3.2018.che/loc/fre/codelists.xml

```diff
@@ -2465,6 +2553,7 @@
       <description>Document papier</description>
     </entry>
   </codelist>
+
```

Comment:
These files comes from gn, iso19115-3, we try to minimise delta even if layout can be made best.